### PR TITLE
`load` cheatcode implementation

### PR DESCRIPTION
**Pull request by MaksymilianDemitraszek**
It provides load cheatcode interface and fixes state copy bug. 

I do not plan any support for data transformer in this release, I may do some related work in the future.

### CODE REVIEW

File: tests/integration/cheatcodes/load/load_test.cairo [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    end
+end
+
+@storage_var
+func target_map_complex_key(a: felt, b: felt) -> (res: Value):
+end
+
+
+@external
+func test_load_in_deployed_contract{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
```
Comment (kasperski95) [inline tests/integration/cheatcodes/load/load_test.cairo:40]:
```suggestion
func test_load_in_user_contract{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
```

**Comment by MaksymilianDemitraszek** [comment]
@mkaput done

File: protostar/starknet/cheatable_state.py [hunk 2]
```diff
@@ -187,6 +212,25 @@ def update_event_selector_to_name_map(
         for selector, name in local_event_selector_to_name_map.items():
             self.event_selector_to_name_map[selector] = name
 
+    def get_abi_with_contract_address(self, contract_address: int) -> AbiType:
```
Comment (kasperski95) [inline protostar/starknet/cheatable_state.py:215]:
```suggestion
    def get_abi_from_contract_address(self, contract_address: int) -> AbiType:
```

File: protostar/commands/test/cheatcodes/load_cheatcode.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        variable_size = self.variable_size(target_contract_address, variable_type)
+
+        if target_contract_address == self.contract_address:
+            return self.load_local(variable_address, variable_size)
+
+        pre_run_contract_carried_state = self.state.contract_states[
+            target_contract_address
+        ]
+        contract_state = pre_run_contract_carried_state.state
+        contract_state.assert_initialized(contract_address=target_contract_address)
```
Comment (kasperski95) [inline protostar/commands/test/cheatcodes/load_cheatcode.py:35]: Can you add a comment explaining this section?

**Review by mkaput — APPROVED** [review]

**Pull request merged**